### PR TITLE
feat(optionsmenu): add disabled variation

### LINKED ISSUE #1962: Add disabled variation of options menu
Let's add a disabled version of the options menu that mimics the dropdown component's disabled state.

This is related to https://github.com/patternfly/patternfly-next/issues/1785

cc @mcarrano

### PR DESCRIPTION
**Pull request by christiemolloy**
fix #1962

### CODE REVIEW

File: src/patternfly/components/OptionsMenu/options-menu.scss [hunk 1]
```diff
@@ -161,6 +162,19 @@
       --pf-c-options-menu__toggle--m-plain--Color: var(--pf-c-options-menu__toggle--m-plain--hover--Color);
     }
   }
+
+  &.pf-m-disabled,
+  &:disabled {
+    pointer-events: none;
+
+    &:not(.pf-m-plain) {
```
Comment (srambach) [inline src/patternfly/components/OptionsMenu/options-menu.scss]: Is there a conscious decision by interaction/design not to change the appearance of a plain options menu?

Comment (christiemolloy) [reply src/patternfly/components/OptionsMenu/options-menu.scss]: @mceledonia @kybaker any thoughts on this? we keep dropdown / options menu / context selector the same visually for the most part. Are you guys ok with having the disabled styling the same?

Comment (mceledonia) [reply src/patternfly/components/OptionsMenu/options-menu.scss]: Yes I would think so! I definitely see dropdown and select as the same thing, to the user a select is just a multiselect dropdown. Context selector feels basically like a dropdown to me too but I'll defer to @kybaker there!

Comment (kybaker) [reply src/patternfly/components/OptionsMenu/options-menu.scss]: @christiemolloy Agree with Michael here I to see them as the same thing.

**Comment by srambach** [comment]
Looks good assuming all the checks pass! bitmoji

**Comment by mattnolting** [comment]
Double checked presentation w/ @kybaker. We're good to go!

**Pull request merged**